src/containers/PipelineRuns/PipelineRuns.test.js

```diff
@@ -362,3 +363,44 @@ it('An invalid filter value is disallowed and reported', async () => {
     )
   ).toBeTruthy();
 });
+
+it('TaskTree handles rerun event in PipelineRuns page', async () => {
+  const mockTestStore = mockStore(testStore);
+  const match = {
+    params: {},
+    url: '/pipelineruns'
+  };
+
+  const onSelect = jest.fn();
+  const { queryByText, getByTestId, getByText } = renderWithRouter(
+    <Provider store={mockTestStore}>
+      <Route
+        path="/pipelineruns"
+        render={props => (
+          <PipelineRunsContainer
+            {...props}
+            match={match}
+            error={null}
+            loading={false}
+            namespace="namespace-1"
```

Comment:
the container should be able to get many of these props from the store so shouldn't need to pass them explicitly in the test